draft-ietf-scitt-architecture.md

```diff
@@ -150,45 +150,6 @@ The core components are based on the Concise Signing and Encryption standard spe
 
 {::boilerplate bcp14-tagged}
```

Comment:
```suggestion
{::boilerplate bcp14-tagged}

- # Use Cases

The building blocks defined in SCITT are intended to support applications in any supply chain that produces or relies upon digital artifacts, from the build and supply of software and IoT devices to advanced manufacturing and food supply.

Detailed use cases are maintained in a separate document: https://datatracker.ietf.org/doc/draft-ietf-scitt-software-use-cases/
```

Reply:
maybe add in the informative reference section of the document header:`I-D.ietf-scitt-software-use-cases: sw-use-cases`
and here: `{{-sw-use-cases}}`